website/content/de/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: TAG Environmental Sustainability
+toc_hide: true
+---
+
+<div class="row mt-5 mb-3">
+    <div class="col-lg-6">
+        <div class="lead">
+        TAG Environmental Sustainability unterstützt Projekte und Initiativen im Zusammenhang mit der Bereitstellung von Cloud Nativen Anwendungen, einschließlich deren Erstellung, Verpackung, Bereitstellung, Verwaltung und Betrieb.
+        </div>
+    </div>
+    <div class="col-lg-6">
+        <img src="/images/tag-environmental-sustainability_color.svg" alt="Tag Environmental Sustainability logo" style="max-width: 300px;">
+    </div>
+</div>
+
+<p class="mt-5 mb-5"><img src="/images/tag-env-sustainability-header.webp" alt="TAG Environmental Sustainability"></p>
+
+Das Ziel dieser TAG ist es, Initiativen zur Umweltfreundlichen Nachhaltigkeit in Cloud Nativen Technologien zu befürworten, zu entwickeln, zu unterstützen und zu evaluieren. Diese TAG wird Werte und mögliche Anreize für Dienstleister identifizieren, um ihren Verbrauch und ihren CO2-Fußabdruck durch den Einsatz von Cloud Nativen Werkzeugen zu reduzieren.
+
+<!-- cSpell:ignore Linktree -->
+
+Alle wichtigen Links für die Umweltschutznachhaltigkeit von TAG sind auch von einer einzigen Linktree-Seite aus verfügbar: [cncfenvtag](https://linktr.ee/cncfenvtag).
+
+- [GitHub Repository](https://github.com/cncf/tag-env-sustainability)
+- [TAG Charter](https://github.com/cncf/tag-env-sustainability/blob/main/charter.md)
+- [Events](https://tag-env-sustainability.cncf.io/events/)
+- Slack-Kanäle:
+  - [#tag-environmental-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6)
+  - [#tag-env-wg-comms](https://cloud-native.slack.com/archives/C068XUD9AEA)
+  - [#tag-env-wg-green-reviews](https://cloud-native.slack.com/archives/C060EDHN431)
+  - [Invite yourself to the CNCF Slack](https://slack.cncf.io/)
```

Comment:
I would maybe translate this to german too

Reply:
Yes. Lets translate this too.

Reply:
Sure